setup.cfg

```diff
@@ -162,6 +162,31 @@ install_requires =
     werkzeug~=1.0, >=1.0.1
     # SQLA still imports the compat
     wtforms<3.0.0
+    # Required by Open Telemetry
```

Comment:
This whole part should be moved to "setup.py" as "[open-telemetry]" extra - you can take a  look there as an example you can take "rabbitmq" extra and find all the places in `setup.py` where `rabbitmq` is mentioned. (`mongodb` is a bit different becaue it uses 'provider` so rabbitmq is better as an example.

https://github.com/apache/airflow/blob/c49d6ec8b67e48d0c0fba1fe30c00f590f88ae65/setup.py#L449